data/examples/declaration/class/associated-data.hs

```diff
@@ -6,15 +6,19 @@ class Foo a where data FooBar a
 class Bar a
   where
     -- | Bar bar
-    data BarBar a
```

Comment:
IIRC I introduced one-line data families because the original examples didn't have that sort of example, only multiline. Why do you want to bring it back?